Foundations/ContentView.swift

```diff
@@ -0,0 +1,46 @@
+//
+//  ContentView.swift
+//  Foundations
+//
+//  Created by José Danilo Silva do Carmo on 14/03/23.
+//
+
+import SwiftUI
+
+struct ContentView: View {
+    private enum Tabs: Hashable {
+        case general, advanced
+    }
+    
+    @State private var idadeCao: String = "";
+    @State private var idadeCao2: Int = 0;
+    
+    var body: some View {
+        TabView {
+            Relationships()
```

Comment:
You need to place the page here, and the tabItem/tag are just a customization and identification